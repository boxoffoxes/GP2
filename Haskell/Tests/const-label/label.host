[
	(n1, empty)
	(n2, 2)
	(n3, empty)
	(n4, 1)
	(n5, empty)
|
]
