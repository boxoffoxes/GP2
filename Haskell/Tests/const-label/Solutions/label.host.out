[
 (n1, empty)
 (n3, empty)
 (n4, 2)
 (n5, empty)
|
]
